scaladoc-js/main/src/searchbar/SearchbarComponent.scala

```diff
@@ -23,9 +23,16 @@ class SearchbarComponent(engine: SearchbarEngine, inkuireEngine: InkuireJSSearch
         Globals.pathToRoot + p.location
       }
 
+      val extensionTargetMessage = if (p.extensionTarget.isEmpty()) {
+        ""
+      } else {
+        " extension on " + p.extensionTarget
+      }
+
       div(cls := "scaladoc-searchbar-row monospace", "result" := "")(
         a(href := location)(
           p.fullName,
+          span(cls := "italic")(extensionTargetMessage),
```

Comment:
Could be just:
```
span(i(extensionTargetMessage))
```
Then, you wouldn't need to add CSS rule to make the text italic.